NEWS.md

```diff
@@ -3,6 +3,7 @@ ClimaLand.jl Release Notes
 
 main
 --------
+- PR[#692] Generalize our forcing ``drivers" to include prescribed soil organic carbon
```

Comment:
This PR will have breaking change right? 
So we should add `![][badge-💥breaking]`

Reply:
yes! let me add that